client/src/components/RequestManagementContainer.tsx

```diff
@@ -91,25 +91,25 @@ class RequestManagementContainer extends Component<{}, RequestManagementContaine
         this.handleCross = this.handleCross.bind(this);
         this.handleReady = this.handleReady.bind(this);
         this.handlePrep = this.handlePrep.bind(this);
-        this.handleDone = this.handleDone.bind(this)
+        this.handleDone = this.handleDone.bind(this);
```

Comment:
you can change class methods to arrow methods if you want to skip `bind`ing everything

Reply:
Really?  WHOA

Reply:
Fixed this in a few files I came across, should check across the project to catch 'em all